packages/connector/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  postgres:
+    image: "postgres" # use latest official postgres version
+    restart: unless-stopped
+    volumes:
+      - database-data:/var/lib/postgresql/data/ # persist data even if container shuts down
+      - ./scripts/init.sh:/docker-entrypoint-initdb.d/init.sh
+    ports:
+      - "5433:5432"
+    environment:
+      POSTGRES_PASSWORD: password
+      POSTGRES_USER: postgres
+      POSTGRES_DB: testing
+  redis:
+    image: "redis"
```

Comment:
what is redis used for?

Reply:
STREAM receipts
https://github.com/coilhq/rafiki/pull/17